061_quadraticEquation.js

```diff
@@ -0,0 +1,32 @@
+// 61. Roots of a quadratic equation
+const quadraticEquation = (a, b, c) => {
+  let descriminanat;
+  let squareroot;
+  let output;
+  discriminanat = (b ** 2 - 4 * a * c);
```

Comment:
spelling mistake
"descriminanat" in the 3rd line , "discriminanat" in the 6th line